### PR TITLE
Install latest runc 1.1.*

### PR DESCRIPTION
**Pull request by cartermckinnon**
**Description of changes:**

Expands `runc` package selection to `1.1.*`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eks-worker-al2-variables.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "pause_container_version": "3.5",
     "pull_cni_from_github": "true",
     "remote_folder": "/tmp",
-    "runc_version": "1.1.5-1.amzn2",
+    "runc_version": "1.1.*",
```
Comment (ravisinha0506) [inline eks-worker-al2-variables.json:26]: What's the version which would get install if we don't add anything here?

Comment (cartermckinnon) [reply eks-worker-al2-variables.json:26]: If we specified `*` here, we would always get the latest. I've scoped it down a bit so we don't install, e.g. `2.0` without warning and break the build.

Comment (cartermckinnon) [reply eks-worker-al2-variables.json:26]: The latest version available is `runc-1.1.7-2.amzn2`, which will be pulled in by `1.1.*` obvs

**Comment by cartermckinnon** [comment]
Oh nice, the doc linter did it's thing 👍 https://github.com/awslabs/amazon-eks-ami/actions/runs/5812932554/job/15759400309?pr=1384

**Review by camrakin — APPROVED** [review]

**Pull request merged**